robotools/evotools/commands.py

```diff
@@ -26,7 +26,7 @@ def evo_make_selection_array(rows: int, columns: int, wells: np.ndarray):
         Number of rows of target labware object
     cols : int
         Number of columns of target labware object
-    wells : List[str]
+    wells
```

Comment:
I think it's not elegant to drop the type hint. Instead, I suggest that you put here `Iterable[str]`
```suggestion
    wells : Iterable[str]
```